### ISSUE TITLE
Escape shuttle hitting asteroids and rendered inoperable

### ISSUE THREAD

**Issue by pyrosshade** [unitystation/unitystation#6724]
This is likely an issue/oversight with other stations as well, perhaps. It's far too easy for escape shuttles to be rendered inoperable from asteroids in the way.

**Comment by corp-0** [comment]
Yes, it is an issue with any map. We're doing some validation to make sure asteroids won't spawn in the way, but it is not perfect. Valid.
